terragrunt/aws/api/lambda.tf

```
@@ -0,0 +1,32 @@
+module "url_shortener_lambda" {
+  source                 = "github.com/cds-snc/terraform-modules?ref=v5.0.0//lambda"
+  name                   = "${var.product_name}-api"
+  billing_tag_value      = var.billing_code
+  ecr_arn                = aws_ecr_repository.api.arn
+  enable_lambda_insights = true
+  image_uri              = "${aws_ecr_repository.api.repository_url}:latest"
+  memory                 = 3008
+  timeout                = 300
+  ephemeral_storage      = 768
+
+  vpc = {
+    security_group_ids = [aws_security_group.api.id]
+    subnet_ids         = module.url_shortener_vpc.private_subnet_ids
```

These will also both need to use the variables rather than the resource properties:
```suggestion
    security_group_ids = [var.api_security_group_id]
    subnet_ids         = var.private_subnet_ids
```

---

Done!